### DISCUSSION TITLE
What are you using for queued jobs?

### DISCUSSION THREAD

**Discussion by wemersonrv** [adonisjs/core#1415] [Help] [answered]
Hello guys, 

What are you using for queued jobs?

I'm using [adonis-kue](https://github.com/nrempel/adonis-kue) for Adonis 4.1, but it has more than 2 years without any update. 

Did you recommend this package yet? Or there is another package that is good for Adonis 4.1 end even compatible with both adonis latest versions? 4.1 and 5 ?

**Comment by msrumon** [comment, marked as answer]
Never heard of this package. I use [bull](https://www.npmjs.com/package/bull), it uses **Redis** to queue the jobs. I have used it w/ Adonis v5 and it works pretty good.

**Reply by wemersonrv** [reply]
> Never heard of this package. I use [bull](https://www.npmjs.com/package/bull), it uses **Redis** to queue the jobs. I have used it w/ Adonis v5 and it works pretty good.

Cool. Thans, i will take a look on this one.

**Reply by popovicieduard** [reply]
@msrumon did you write any package for it or are you directly using the bull package ?

**Comment by wemersonrv** [comment]
Found this package that made It simple. Cool!

https://github.com/Rocketseat/adonis-bull

**Reply by RodolfoSilva** [reply]
Hi @wemersonrv 
I started a PR to add support for Adonis 5, take a look https://github.com/Rocketseat/adonis-bull/pull/39

**Reply by ashokgelal** [reply]
@RodolfoSilva Thanks for the PR. I was looking for Adonis5 Bull package and ran into this discussion. Couple of notes:
1. How do one configure Redis connections? It looks like you removed `config/bull.js` from the README. Not sure if that was intentional or not.
2. I'd suggest sending a different PR for Docker and eslint configs. It pollutes the main purpose of the PR.

Thanks again!

**Reply by RodolfoSilva** [reply]
Thanks @ashokgelal 😁

**Reply by ashokgelal** [reply]
No, thank YOU, @RodolfoSilva. I'm looking forward to getting the PR merged and using the package. You should also write an article on how to use it with an example :) I'm sure many people are looking for how to use a queue in an Adonis project.

**Reply by RodolfoSilva** [reply]
I'll try write this week 😜. Thanks

**Reply by jpedroschmitz** [reply]
Hey, I'm a maintainer from [@rocketseat/adonis-bull](https://github.com/Rocketseat/adonis-bull) and we have released the next version of the library, which is ready for Adonis v5. You can find the [documentation here](https://github.com/Rocketseat/adonis-bull/tree/alpha#readme) 👊🏻 

We still support Adonis v4. Version 5 support is under the alpha tag on NPM.

And of course, special thanks to @RodolfoSilva, for starting the migration to v5 💜
